src/vos/vos_pool.c

```diff
@@ -913,6 +918,18 @@ vos_pool_ctl(daos_handle_t poh, enum vos_pool_opc opc)
 		if (pool->vp_vea_info != NULL)
 			vea_flush(pool->vp_vea_info, false);
 		break;
+	case VOS_PO_CTL_SET_POLICY:
+		if (param == NULL)
+			return -DER_INVAL;
+
+		struct policy_desc_t *p = (struct policy_desc_t *)param;
```

Comment:
I'm slightly surprised this compiles as I thought we didn't allow mixing of declarations and code, but what I really wanted to say is there's no cast needed here, param is a void * which will alias to anything.